components/storage/ComparisonCellContent.vue

```diff
@@ -20,6 +20,45 @@
         ]"
         ><i class="mdi mdi-speedometer"
       /></span>
+      <span
+        v-else-if="
+          typeof feature.class === 'string' && feature.class.startsWith('slow')
```

Comment:
does this need to be a `startsWith` comparison? the `fast` one above is because of `fast, faster, fastest` - will there be varying degrees of slow?

Reply:
Yep, agreed! Fixed! Thank you, @mcmcgrath13!!